minestudio/inference/generator/mine_generator.py

```
@@ -6,8 +6,11 @@
 '''
 import os
 import ray
+
 from typing import Callable, Optional, List, Dict, Tuple, Literal, Generator
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)